api/tests/unit/app_analytics/test_unit_app_analytics_views.py

```diff
@@ -131,3 +146,88 @@ def test_get_total_usage_count_for_non_admin_user_returns_403(
 
     # Then
     assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.skipif(
+    "analytics" not in settings.DATABASES,
+    reason="Skip test if analytics DB is not configured",
+)
+@pytest.mark.django_db(databases=["default", "analytics"])
+def test_set_sdk_analytics_flags_with_identifier(
+    api_client: APIClient,
+    environment: Environment,
+    feature: Feature,
+    identity: Identity,
+    settings: SettingsWrapper,
+) -> None:
+    # Given
+    settings.USE_POSTGRES_FOR_ANALYTICS = True
+    url = reverse("api-v2:analytics-flags")
+    api_client.credentials(HTTP_X_ENVIRONMENT_KEY=environment.api_key)
+    feature_request_count = 2
+    data = {
+        "evaluations": [
+            {
+                "feature_name": feature.name,
+                "identity_identifier": identity.identifier,
```

Comment:
As per the above, I'd like to see a test where the `identity_identifier` is omitted.

Reply:
This is done.